fuzzing/fuzzer_worker.go

```diff
@@ -291,6 +291,14 @@ func (fw *FuzzerWorker) testNextCallSequence() (calls.CallSequence, []ShrinkCall
 		// Update our metrics
 		fw.workerMetrics().callsTested.Add(fw.workerMetrics().callsTested, big.NewInt(1))
 
+		// If we reached our transaction threshold, halt
+		callsTested := fw.fuzzer.metrics.CallsTested()
+		testLimit := fw.fuzzer.config.Fuzzing.TestLimit
+		if testLimit > 0 && (!callsTested.IsUint64() || callsTested.Uint64() >= testLimit) {
+			fw.fuzzer.logger.Info("Transaction test limit reached, halting now...")
+			fw.fuzzer.Stop()
+		}
+
```

Comment:
https://github.com/crytic/medusa/pull/364/commits/c93b5579567b8f4882dca113110f95f8003d4a70 can probably be reverted but it made it easier to eyeball the hit count e.g. 1000 runs on a contract with `f() { revert{}` should have a hit count of 1000 reverts